advanced_flask/templates/home.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block main_info%}
+<div>
+    <ul>
+        <li>
+            <a href="{{url_for('products.get_products')}}">Products</a>
+        </li>
+        <li>
+            <a href="{{url_for('supermarkets.get_supermarkets')}}">Supermarkets</a>
+        </li>
+    </ul>
+</div>
+{% endblock %}
+
```

Comment:
too much blank lines. You need to leave only one empty line at the bottom of every file